README.md

```diff
@@ -15,6 +15,306 @@ This repository contains two smart contracts:
 - [**Cert Wallet Contract**](https://github.com/kommitters/chaincerts_governance/tree/main/certs_wallet):
   The `cert_wallet` contract stores the Chaincerts of a specific user, and each user must have their own `cert_wallet`. This contract allows users to view their Chaincerts, add organizations authorized to issue certificates to them, and execute distribution and revocation actions on their certificates. Only functions that can be executed by the `cert_governance` contract can modify certificates in a `cert_wallet`.
 
+## System overview
+The following image illustrates how the smart contracts manage the certificate governance process.
+![Flow of the system](https://user-images.githubusercontent.com/89233604/232091954-1aff8267-4f0d-4f01-8c40-775fa91cd397.png)
+
+## Contracts workflow
+1. In order to deploy the contracts, we require an account that can be created on the Stellar laboratory using the Friendbot tool.
+2. Two critical aspects are involved in this process: the wallet contract, which enables users to hold and manage their certificates, and the governance contract, which allows organizations to issue, verify, and distribute certificates to user wallets.
+3. Both contracts need to be initialized to add the fundamental configuration to them.
+4. To enable organizations to distribute certificates to the wallet, wallets must include these organizations on their access control list for authorization purposes.
+5. Once a wallet authorizes an organization, it can proceed with certificate distribution using the governance contract.
+    * Organizations can revoke certificates as well, provided that revocability is allowed by the governance contract.
+	* Certificates can be set to expire if an expiration_time is specified during the contract initialization.
+	* Revocation of a certificate is only permissible by the organization that distributed it.
+
+
+## Pre-requirements
+To work with the contracts, Rust and Soroban need to be installed. The process for installation is detailed in the [Soroban setup](https://soroban.stellar.org/docs/getting-started/setup).
+
+## Setup
+``` 
+# Clone the repository 
+git clone git@github.com:kommitters/chaincerts_governance.git
+
+# Build the project and get dependencies 
+cd chaincerts_governance
+cargo build
+ ```
+
+## Testing
+To test the contract run `cargo test -- --show-output` or ` cargo test -- --nocapture`
```

Comment:
Maybe the second way for running the tests is unnecessary, since is more focused on printing stuff with the std:println macro, what do you think? 🤔 
```suggestion
To test the contract run `cargo test -- --show-output`
```